### PR TITLE
feat(src): add multi-line text input

### PR DESCRIPTION
**Pull request by stonewallstan**

### CODE REVIEW

File: src/reducers/auth.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import * as types from '../actions/types';
 
 const INITIAL_STATE = {
-  authenticated: false,
+  authenticated: true,
```
Comment (Jumer-Caragay) [inline src/reducers/auth.js]:
```suggestion
  authenticated: false,
```
Reset this back to false

File: src/components/Questionnaire/Submission.jsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ import { submitApp } from '../../actions/index';
 class Submission extends Component {
   submitApplication(form) {
     const { submitApp: submitApplication } = this.props;
+    console.log(form);
```
Comment (Jumer-Caragay) [inline src/components/Questionnaire/Submission.jsx]: Remove the console log.

File: src/index.jsx [hunk 3]
```diff
@@ -22,11 +23,12 @@ ReactDOM.render(
     <HashRouter>
       <div>
         <Switch>
+          <Route path={['/', '/login']} exact component={Login} />
```
Comment (Jumer-Caragay) [inline src/index.jsx]: Currently PropTypes throws an error when you send an array of strings for the path, consider using a regex expression for the path instead.

Comment (mianuddin) [reply src/index.jsx]: You should address this in the other PR, this code shouldn't be here in the first place.

**Comment by stonewallstan** [comment]
Closing this PR as it was put on the wrong branch. The fixed PR is located [here](https://github.com/slohacks/attendee-application-front-end/pull/20).

**Pull request closed without merging**